doc/node.1

```diff
@@ -1,28 +1,54 @@
 .TH NODE 1 2016 Node.js Node.js
 
-.\ This is a man page comment.
-.\ Man page syntax (actually troff syntax) is somewhat obscure, but the
-.\ important part is is that .<letter> specifies <letter>'s syntax for that
-.\ line, and \f<letter> specifies it for the characters that follow.
-.\ See http://liw.fi/manpages/ for more info.
+.\" This is a man page comment.
+.
+.\" Man page syntax (actually troff syntax) is somewhat obscure, but the
+.\" important part is is that .<letter> specifies <letter>'s syntax for that
+.\" line, and \f<letter> specifies it for the characters that follow.
+.
+.\" .B   Bold line
+.\" .I   Italic line (Rendered as underlined text in terminals)
+.\" .BI  Alternating bold/italics without spaces between arguments.
+.\"      Use `\ ` to include an "unpaddable" (literal) space in the output.
+.\" .RI  Alternating roman/italic
+.
+.\" See http://liw.fi/manpages/ for an overview, or http://www.troff.org/54.pdf
+.\" for detailed language reference.
+.de ur
```

Comment:
I'm not sure what's being asked here... is this concerning the first or second line?

1. I felt impelled to offer readers a link to a more authoritative resource on the language, as the former link, while helpful, is extremely limited and only covers very rudimentary aspects of the language.

2. The macro is simply a cleaner way of formatting the [links below](https://github.com/nodejs/node/pull/7819/files#diff-176884dfbb649efd3b1316139eb4cd69R201). Ordinarily I would've used the `.UR/.UE` macros provided by Groff's `an-ext.tmac`, but I'm aware that this isn't available on all systems. I wanted to guarantee this manpage would pose no portability issues. Which is also the reason I named the macro `.ur` instead of `.url`, as classical troff didn't permit macro names longer than 2 characters.